pkgs/applications/radio/yaac/default.nix

```diff
@@ -0,0 +1,34 @@
+{ makeWrapper, lib, stdenv, fetchurl, unzip, jre8 }:
+
+stdenv.mkDerivation rec {
+  version = "1.0-beta181";
+  pname = "YAAC";
+
+  src = fetchurl {
+    url = "https://www.ka2ddo.org/ka2ddo/YAAC.zip";
+    sha256 = "564071bb51fb4a91850ff53ff7d0c9477c1715d7b11eed9e97c89ead9df7332b";
```

Comment:
Please use SRI hash here.
> Nixpkgs contributors are currently recommended to use hash.

https://nixos.org/manual/nixpkgs/stable/#fetchurl

Reply:
Updated in latest.